### DISCUSSION TITLE
Adding in OAuth Support to PocketHost

### DISCUSSION THREAD

**Discussion by cuire** [pockethost/pockethost#316] [Q&A] [answered]
The new webapp does not provide the ability to login via Github. I had an old oauth account. How do I access it? Creating account with the same email does not work. The instance is still running, but I cant update it.
If this functionality is in roadmap, I can make a PR

**Comment by benallfree** [comment, marked as answer]
Hi, we never had Github SSO. Were you thinking of a different project?

By the way, if you do want to help us implement oAuth providers, I'd be happy to get you started :)

**Reply by cuire** [reply]
My memory sometimes fails me. I remembered that I used Github, double-checked and found credentials 😬

**Comment by brewhousedigital** [comment]
@benallfree We had previously talked about adding Discord OAuth. What are your thoughts on adding Google and Github support as well? I see Github as a really easy to use one since a lot of other developer-oriented tools utilize it (Vercel / Netlify). 

As for the UI, we use SvelteKit and FontAwesome for our Icons. They have support for all three of those providers.

```html
<i class="fa-brands fa-github"></i>
<i class="fa-brands fa-discord"></i>
<i class="fa-brands fa-google"></i>
```

**Comment by cuire** [comment]
There is one problem. Pockethost supports FTP accses of instances, the creditnails for which is the login and password from the app. But in the case of oauth this data is not available.
